recipe/pre-unlink.bat

```diff
@@ -1,3 +1,3 @@
 @echo off
 if "%CONDA_PREFIX%"=="" (set _PREFIX="%PREFIX%") else (set "_PREFIX=%CONDA_PREFIX%")
-"%_PREFIX%\python.exe" -m conda tos clean --all >>"%_PREFIX%\.messages.txt" 2>&1
+"%_PREFIX%\python.exe" -m conda tos backup --max-age 604800 --all-locations >>"%_PREFIX%\.messages.txt" 2>&1
```

Comment:
I think the `clean` command still needs to be run after `backup` since `backup` does not delete the ToS files.